pkg/local_object_storage/pilorama/forest_test.go

```diff
@@ -202,6 +204,19 @@ func testForestTreeDrop(t *testing.T, s Forest) {
 
 	_, err = s.TreeGetByPath(cid, trees[1], AttributeFilename, []string{"path"}, true)
 	require.NoError(t, err)
+
+	for i := range trees {
+		_, err := s.TreeAddByPath(d, trees[i], AttributeFilename, []string{"path"},
+			[]KeyValue{{Key: "TreeName", Value: []byte(trees[i])}})
+		require.NoError(t, err)
+	}
+	list, err := s.TreeList(cid)
```

Comment:
check here that the `list` is not empty?

Reply:
Thanks, fixed.